content/design-systems-for-developers/react/en/test.md

```diff
@@ -89,7 +89,7 @@ Visual testing helps identify UI changes in Storybook. Review the changes to con
 
 Let’s add visual testing to the continuous integration job. Open `.circleci/config.yml` and add the test command.
 
-```yml
+```yaml
```

Comment:
I think there is something about how we are using the library that it required the full name. We have to tell prism js which languages we want to use so we don't have to load them all and I manually said `yaml` so maybe the library requires that you reference it in that way from there on out.